crates/aptos-rest-client/src/lib.rs

```diff
@@ -1292,7 +1320,7 @@ impl Client {
         &self,
         address: AccountAddress,
     ) -> AptosResult<Response<AccountResource>> {
-        let url = self.build_path(&format!("accounts/{}", address))?;
+        let url = self.build_path(&format!("accounts/{}", address.to_hex()))?;
```

Comment:
These should probably not be `.to_hex()` if it should be accepting whatever any address input on the API.

Reply:
Perhaps, but there are lots of things like this in the PR. I'd rather make such changes in later PRs.